src/components/branding/branding.constant.ts

```diff
@@ -42,13 +42,14 @@ export const jsonBranding = JSON.stringify({
   'docs': {
     'devfile': 'https://www.eclipse.org/che/docs/che-7/making-a-workspace-portable-using-a-devfile/',
     'workspace': 'https://www.eclipse.org/che/docs/che-7/workspaces-overview/',
-    'factory': 'https://www.eclipse.org/che/docs/factories-getting-started.html',
-    'organization': 'https://www.eclipse.org/che/docs/organizations.html',
-    'converting': 'https://www.eclipse.org/che/docs/che-7/converting-a-che-6-workspace-to-a-che-7-devfile/',
+    'factory': 'https://www.eclipse.org/che/docs/che-7',
+    'organization': 'https://www.eclipse.org/che/docs/che-7',
+    'converting': 'https://www.eclipse.org/che/docs/che-7',
     'certificate': 'https://www.eclipse.org/che/docs/che-7/importing-certificates-to-browsers/',
     'general': 'https://www.eclipse.org/che/docs/che-7',
     'storageTypes': 'https://www.eclipse.org/che/docs/che-7/configuring-storage-types/',
-    'webSocketTroubleshooting': 'https://www.eclipse.org/che/docs/che-7/troubleshooting-network-problems/#troubleshooting-websocket-secure-connections_troubleshooting-network-problems'
+    'webSocketTroubleshooting': 'https://www.eclipse.org/che/docs/che-7/troubleshooting-network-problems/#troubleshooting-websocket-secure-connections_troubleshooting-network-problems',
+    'faq': 'https://www.eclipse.org/che/docs/che-7'
```

Comment:
I thought it's optional... Do we have faq in upstream? Don't we have faq section in docs?

Reply:
It's not overridden in CRW https://github.com/redhat-developer/codeready-workspaces/blob/crw-2.5-rhel-8/assembly/branding/product.json.template

Reply:
you're right, it's optional and adding this line makes FAQ button to appear. I'm removing this line.

Reply:
I assume this file needs to be regenerated